Public/Src/Deployment/privatePackages.dsc

```diff
@@ -121,6 +121,12 @@ namespace PrivatePackages {
                         importFrom("BuildXL.Utilities.Instrumentation").withQualifier(winx64Qualifier).Common.dll,
                     ],
                 },
+                {
+                    subfolder: r`runtimes/win-x64/native/`,
+                    contents: [
+                        importFrom("BuildXL.Utilities").withQualifier(winx64Qualifier).Native.dll.nativeContent,

```

Comment:
If this is the only reason why you need this "nativeContent", I suggest specifying it directly as a top-level value in `Public/Src/Utilities/Native/BuildXL.Native.dsc` (instead of modifying the `ManagedAssembly` type and plumbing that change through the rest of the SDKs).

Concretely, in `Public/Src/Utilities/Native/BuildXL.Native.dsc` add something like
```typescript
  @@public
  export const nativeWin = [ importFrom("BuildXL.Sandbox.Windows").Deployment.natives ];

  @@public
  export const nativeMac = [
     MacServices.Deployment.sandboxMonitor,
     MacServices.Deployment.ariaLibrary,
     MacServices.Deployment.interopLibrary
  ];
```